Update broadcast.mdx (#40627)
supabase/supabase: apps/docs/content/guides/realtime/broadcast.mdx

</Admonition>

> For Authorization we insert a message and try to read it and rollback the transaction to verify that the RLS policies set by the user are being respected by the user joining the channel but this messages won't be sent to the user. You can read more about it in the [Authorization](/docs/guides/realtime/authorization) docs

Reviewer: ```suggestion
For Authorization we insert a message and try to read it and rollback the transaction to verify that the RLS policies set by the user are being respected by the user joining the channel but this messages won't be sent to the user. You can read more about it in the [Authorization docs](/docs/guides/realtime/authorization).
```

> For Authorization we do insert a message and try to read it and then we echo it back as way to verify that the RLS policies set by the user are being respected by the user joining the channel but this messages won't be sent to the user. You can read more about it in the [Authorization](/docs/guides/realtime/authorization) docs

Reviewer: OK then, I guess I don't fully understand the context.

```suggestion
For Authorization we insert a message and try to read it and rollback the transaction to verify that the RLS policies set by the user are being respected by the user joining the channel but this messages won't be sent to the user. You can read more about it in the [Authorization](/docs/guides/realtime/authorization) docs
```
